### PR TITLE
Test coverage improvement for inmemory/*

### PR DESCRIPTION
**Pull request by andrewhzy**
test coverage improved to 100% for inmemory/*, except inmemory/list.go which is included in PR #183

### CODE REVIEW

File: inmemory/zset_test.go [hunk 1]
```diff
@@ -70,8 +74,28 @@ func TestDB_ZMembers(t *testing.T) {
...
+	for _, test := range tests {
+		card, err := testDB.ZCard(test.bkt)
+		assertions.Equal(test.num, card)
+		assertions.Equal(test.err, err)
+	}
+}
+
 func TestDB_ZCount(t *testing.T) {
 	bucket, _, _, _ := initZAddItems()
+	num, er := testDB.ZCount("neBucket", 11, 12, nil)
```
Comment (xujiajun) [inline inmemory/zset_test.go:97]: [golangci] reported by reviewdog 🐶 ineffectual assignment to num (ineffassign)

Comment (xujiajun) [reply inmemory/zset_test.go:97]: Instead of using 'er' for error variables, you can use 'err'

File: inmemory/set.go [hunk 2]
```diff
@@ -38,6 +38,9 @@ func (db *DB) SRem(bucket string, key string, items ...[]byte) error {
 		if _, ok := shardDB.SetIdx[bucket]; !ok {
 			return nutsdb.ErrBucket
 		}
+		if items == nil {
+			items = make([][]uint8, 1)
```
Comment (xujiajun) [inline inmemory/set.go:42]: ？ I don't understand what this code means

**Review by andrewhzy — COMMENTED** [review]
as empty string is valid  as key for both map and redis, I updated ds/set/set.go to allow it, see line 63.

**Review by xujiajun — APPROVED** [review]

**Pull request merged**